orca-redis/src/main/java/com/netflix/spinnaker/orca/pipeline/persistence/jedis/RedisExecutionRepository.java

```diff
@@ -695,7 +695,7 @@ public PipelineExecution retrieveByCorrelationId(
   @Override
   public PipelineExecution retrievePipelineForCorrelationId(@Nonnull String correlationId)
       throws ExecutionNotFoundException {
-    String key = format("pipelineCorrelation:%s", correlationId);
+    String key = format("correlation:%s", correlationId);
```

Comment:
This seems like a pretty big change.  Why do we need it?

Reply:
In the absence of any existing code that generates keys predicated upon the prefix `pipelineCorrelation`, it follows that `pipelineCorrelation` will not be present within our database. Given that pipeline and orchestration executions are endowed with unique identifiers, we may opt to employ the same key prefix for both.

Reply:
https://github.com/spinnaker/orca/blob/6dfe7ddca68d4650eea3010b2736804be7a30cbd/orca-redis/src/main/java/com/netflix/spinnaker/orca/pipeline/persistence/jedis/RedisExecutionRepository.java#L1366